Disher/View Controllers/ListContentViewController.h

```diff
@@ -12,7 +12,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @protocol ListContentDelegate<NSObject>
-- (void) didUpdateName:(NSString *)name;
+- (void) didUpdateName:(NSString *)name withImage:(UIImage *)image;
```

Comment:
should we rename this from `didUpdateName` if it's for both the name and image?

Reply:
It takes two parameters so I just named it didUpdateNameWithImage; objective-c is weird